.claude/commands/clonefix.md

```
@@ -0,0 +1,418 @@
+     - **Report to user**: List remaining conflicted files
+     - **Request manual intervention**: "Cannot auto-resolve conflicts in: {files}"
+     - **HALT workflow**: Do NOT proceed to Phase 2 until resolved
+   - **Save proof of resolved files**: `git diff --name-only --cached > /tmp/{repo_name}_clonefix_proof/conflicts_resolved.txt`
+6. **Commit conflict resolution** (only if conflicts were auto-resolved in step 5):
+   ```bash
+   git commit -m "resolve: Auto-resolve merge conflicts with {base_branch}
+
+🤖 Generated by /clonefix command
+
+Co-Authored-By: Claude <noreply@anthropic.com>"
+   ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)